content/tools/plugins/JenkinsPlugin.md

```diff
@@ -13,44 +13,44 @@ tags: "tools Jenkins SDK Integration Java"
 
 ## Getting Started - Defining a Connection
 
-1. In your Jenkins dashboard, go to **Manage Jenkins** in the left sidebar. 
+* In your Jenkins dashboard, go to **Manage Jenkins** in the left sidebar. 
```

Comment:
@contrast-product Changed these back to unordered list as the numbering is not rendered properly (see image). 
![image](https://user-images.githubusercontent.com/17487428/71483042-e44c2680-27d3-11ea-9622-55c479edc517.png)